antora/docs/modules/ROOT/pages/index.adoc

```diff
@@ -14,7 +14,7 @@ xref:pipeline_policy.adoc[Pipeline Policy].
 == Additional Documentation
 
 * https://redhat-appstudio.github.io/docs.stonesoup.io/[Stonesoup Documentation]
```

Comment:
It's not part of this PR, but should we just delete the stonesoup reference?

Reply:
Easy fix, while we're doing housekeeping...